src/proto/Polkadot.proto

```diff
@@ -68,11 +70,11 @@ message SigningInput {
     bytes genesis_hash = 2;
     uint64 nonce = 3;
     uint32 spec_version = 4;
-    bytes tip = 5;  // big integer
-    Era era = 6;    // empty means Immortal
-    bytes private_key = 7;
-    Network network = 8;
-    uint32 extrinsic_version = 9;
+    uint32 transaction_version = 5;
+    bytes tip = 6;  // big integer
```

Comment:
what is tip?

Reply:
Tips are an optional transaction fee that users can add to give a transaction higher priority.